target/bin/updatemailuser

```diff
@@ -14,15 +14,15 @@ USER="${1}"
 shift
 PASSWD="${*}"
 
-function usage  {	echo "Usage: updatemailuser <user@domain.tld> [password]" ; }
+function __usage { echo 'Usage: updatemailuser <user@domain.tld> [password]' ; }
 
-[[ -z ${USER} ]] && { usage ; _exit_with_error "no username specified" ; }
+[[ -z ${USER} ]] && { __usage ; _exit_with_error 'No username specified' ; }
```

Comment:
```suggestion
[[ -z ${USER} ]] && { _exit_with_error 'No username specified' ; }
```

Reply:
As I said in the comment above, I'd always log the usage in such cases.

Reply:
Wrong assumption form my side. I agree.